run_kraken.py

```diff
@@ -348,21 +350,16 @@ def main(cfg):
 
                         # Check for critical alerts when enabled
                         if check_critical_alerts:
-                            logging.info("Checking for critical alerts firing post choas")
-
-                            ##PROM
-                            query = r"""ALERTS{severity="critical"}"""
-                            end_time = datetime.datetime.now()
-                            critical_alerts = prometheus.process_query(
-                                query
-                            )
-                            critical_alerts_count = len(critical_alerts)
-                            if critical_alerts_count > 0:
-                                logging.error("Critical alerts are firing: %s", critical_alerts)
-                                logging.error("Please check, exiting")
+                            prometheus_plugin.critical_alerts(prometheus,
```

Comment:
moved the critical alert collection to the prometheus plugin to not pollute too much the `run_kraken.py`